src/cascadia/TerminalSettingsModel/AppearanceConfig.cpp

```diff
@@ -156,3 +171,24 @@ winrt::hstring AppearanceConfig::ExpandedBackgroundImagePath()
         return winrt::hstring{ wil::ExpandEnvironmentStringsW<std::wstring>(path.c_str()) };
     }
 }
+
+void AppearanceConfig::_logSettingSet(const std::string_view& setting)
+{
+    _changeLog.emplace(setting);
+}
+
+void AppearanceConfig::_logSettingIfSet(const std::string_view& setting, const bool isSet)
+{
+    if (isSet)
+    {
+        _logSettingSet(std::string{ setting });

```

Comment:
you still don't need to construct a string here. you're taking a string_view, constructing a string, then passing it as a string view again!

```suggestion
        _logSettingSet(setting);
```

Reply:
And then, it's copying it as a string again!